Lib/unittest/mock.py

```diff
@@ -1601,7 +1601,11 @@ def _patch_object(
     When used as a class decorator `patch.object` honours `patch.TEST_PREFIX`
     for choosing which methods to wrap.
     """
+    if type(target) is str:
+        raise TypeError(f"{target!r} must be the actual object to be patched, "
+                        f"not a str")
```

Comment:
This one's just personal taste, but I'd prefer to keep this string together (and especially avoid the unnecessary second f-string):

```py
        raise TypeError(
            f"{target!r} must be the actual object to be patched, not a str"
        )
```

Reply:
Could you explain what's wrong with having two f-strings? To know for future.

Reply:
My main gripe was that the second string didn't need to be an f-string (there was nothing to be formatted). It's purely a stylistic choice, it all compiles the same!